src/pudl/metadata/codes.py

```diff
@@ -1231,4 +1231,256 @@
+                ),
+                ("PSEI", "pugent_sound_energy", "Puget Sound Energy"),
+                (
+                    "SC",
+                    "south_carolina_public_service",
+                    "South Carolina Public Service Authority",
+                ),
+                (
+                    "SCEG",
+                    "south_carolina_electric_and_gas",
+                    "South Carolina Electric & Gas Company",
+                ),
+                ("SCL", "seattle_city_light", "Seattle City Light"),
+                ("SEC", "seminole_electric_coop", "Seminole Electric Cooperative"),
+                (
+                    "SECI",
+                    "sunflower_electric_power",
+                    "Sunflower Electric Power Corporation",
+                ),
+                ("SEPA", "southeastern_power", "Southeastern Power Administration"),
+                (
+                    "SOCO",
+                    "southern_company_services",
+                    "Southern Company Services, Inc. - Trans",
+                ),
+                ("SPA", "southwestern_power", "Southwestern Power Administration"),
+                (
+                    "SPRM",
+                    "city_utilities_of_springfield",
+                    "City Utilities Of Springfield, MO",
+                ),
+                (
+                    "SPS",
+                    "southwestern_public_service",
+                    "Southwestern Public Service Co. (Xcel Energy)",
+                ),
+                ("SRP", "salt_river_project", "Salt River Project"),
+                ("SWPP", "southwest_power_pool", "Southwest Power Pool"),
+                ("TAL", "city_of_tallahassee", "City of Tallahassee"),
+                ("TEC", "tampa_electric", "Tampa Electric Company"),
+                ("TEPC", "tuscon_electric_power", "Tucson Electric Power Company"),
+                ("TIDC", "turlock_irrigation_district", "Turlock Irrigation District"),
+                (
+                    "TPWR",
+                    "city_of_tacoma",
+                    "City of Tacoma, Department of Public Utilities, Light Division",
+                ),
+                ("TVA", "tennessee_valley_authority", "Tennessee Valley Authority"),
+                (
+                    "WACM",
+                    "western_area_power_mountain",
+                    "Western Area Power Administration - Rocky Mountain Region",
+                ),
+                (
+                    "WALC",
+                    "western_area_power_southwest",
+                    "Western Area Power Administration - Desert Southwest Region",
+                ),
+                (
+                    "WAUE",
+                    "western_area_power_east",
+                    "Western Area Power Administration - Upper Great Plains East",
+                ),
+                (
+                    "WAUW",
+                    "wester_area_power_west",
```

Comment:
Misspelling in the label. Missing the last N in western

Reply:
ty!